lib/src/blurhash.dart

```diff
@@ -177,10 +192,14 @@ String encodeBlurHash(
   int width,
   int height, {
   int numCompX = 4,
-  int numpCompY = 3,
+  int numCompY = 3,
```

Comment:
I know the typo is really annoying, but renaming the parameter is technically a breaking change, which I really don't want to do for a deprecated method.